pro/src/screens/OfferIndividual/StocksEvent/StockFormList/StockFormList.tsx

```diff
@@ -75,8 +75,9 @@ const StockFormList = ({ offer, onDeleteStock }: IStockFormListProps) => {
                     callback: async () => {
                       if (stockValues.stockId) {
                         if (
-                          parseInt(stockValues.bookingsQuantity, 10) > 0 ||
-                          mode === OFFER_WIZARD_MODE.EDITION
+                          (mode === OFFER_WIZARD_MODE.EDITION ||
+                            stockValues.bookingsQuantity !== '0') &&
+                          stockValues.stockId !== undefined
```

Comment:
Je comprends pas trop ce que tu fix ici, le if(stockValues.stockId) ligne 76 empeche qu'on arrive ici avec une valeur undefined non?

Reply:
Yes c'est normal, si on ne met pas cette condition, lorsqu'on supprime le dernier stock, une ligne de stock est toujours disponible alors qu'il n'y a aucun stock id (normal), et si on clique sur les boutons d'action et qu'on est en edition, la pop in s'affiche, et rien ne se passe lorsque l'utilisateur clique sur le bouton supprimer. (donc cette condition permet de ne pas afficher la pop in s'il n'y a aucun stock déjà créé)